can_messages.asciipb

```diff
@@ -174,18 +174,6 @@ msg {
 
 msg {
   id: 26
```

Comment:
Might want to reserve a "steering angle" message purely for telemetry since apparently people want to support it. From a firmware perspective, it's been downgraded to a unimportant metric.